models/user.js

```diff
@@ -74,21 +74,21 @@ exports.findBy = async property => {
   const key = Object.keys(property)[0];
   let findByQuery;
   switch (key) {
-    case 'firstName':
-      findByQuery = 'SELECT * FROM "users" WHERE "firstName" = $1 LIMIT 1';
-      break;
-    case 'lastName':
-      findByQuery = 'SELECT * FROM "users" WHERE "lastName" = $1 LIMIT 1';
-      break;
-    case 'email':
-      findByQuery = 'SELECT * FROM "users" WHERE "email" = $1 LIMIT 1';
-      break;
-    case 'birthYear':
-      findByQuery = 'SELECT * FROM "users" WHERE "birthYear" = $1 LIMIT 1';
-      break;
-    case 'student':
-      findByQuery = 'SELECT * FROM "users" WHERE "student" = $1 LIMIT 1';
-      break;
+  case 'firstName':
```

Comment:
we should indent here

Reply:
linter no likey 🤷‍♀️ 

do you feel strongly enough about that to edit the linting file? I'm not sure I do 🤔